### DISCUSSION TITLE
Hubs Cloud - Number Of Rooms Limit

### DISCUSSION THREAD

**Discussion by ericborgos** [mozilla/hubs#5723] [Hubs Cloud Questions]
Is there any limit to the number of rooms I can create on 1 server? I know each room shouldn't have more than 25 users at once, but could I for example have 25 rooms with 1 user each? Could I have 250 rooms where each room only gets used a few times a day (not usually at the same time as the others)?

**Comment by pattersonbl2** [comment]
Currently, you can have 25 people in a room.  You may be able to set up 2 rooms with 50 people but you may run into performance issues.
